src/organizations/components/OrgProfileTab/SuspendPaidOrgOverlay.tsx

```diff
@@ -158,6 +160,16 @@ export const SuspendPaidOrgOverlay: FC = () => {
         >
           You will be able to recover this organization's data for up to 7 days
           by contacting support. It will be unrecoverable afterwards.
+          {params.users > 0 && (
+            <>
```

Comment:
Is it `params?.users > 0` or `params?.users > 1`? If there is only one user on the org (the one removing it), I don't think we need to warn the user.

Reply:
ohh I think I assumed users.length === 0 meant org had no other users aside from the user who's currently trying to delete it. If that's not the case, yes, we should definitely update the code to say `params?.users > 1`. I'll get that fixed